traveler-swift-core/TravelerKitTests/Mocking/DataResponses.swift

```diff
@@ -0,0 +1,380 @@
+            "title": "Top Rated by travellers",
+            "subTitle": nil,
+            "description": "Top Rated by travellers",
+            "featured": false,
+            "items": productItemsArray
+        ]
+
+        let groupsJSON: MockJSON = [
+            "groups": [groupQuery,groupItems]
+        ]
+
+        guard let data = groupsJSON.jsonString()?.data(using: .utf8) else {
+            fatalError("Bad JSON for Catalog")
+        }
+
+        return data
+    }
+
+    //MARK:- Supplier (No Trademark)
+    static func supplierDataNoTradeMark() -> Data {
+        //given
+        let incompleteTrademarkJSON: MockJSON = [
+            "id": "SupplierID",
+            "name": "Tiqets",
+            "trademark": nil
+        ]
+
+        guard let data = incompleteTrademarkJSON.jsonString()?.data(using: .utf8) else {
+            fatalError("Bad JSON for Supplier")
+        }
+
+        return data
+    }
+
+    //MARK:- Supplier
+    static func supplierData() -> Data {
+        let trademarkJSON: MockJSON = [
+            "iconUrl": "https://myicon.com",
+            "copyRight":"Simple ricks"
+        ]
+
+        let supplierJSON: MockJSON = [
+            "id": "SuppliedID",
+            "name": "Tiqets",
+            "trademark": trademarkJSON
+        ]
+
+        guard let data = supplierJSON.jsonString()?.data(using: .utf8) else {
+            fatalError("Bad JSON for Supplier")
+        }
+        return data
+    }
+    
+    //MARK:- Trademark (Bad URL)
+    static func trademarkWithBadUrl() -> Data {
+        let tradeMarkJSON: MockJSON = [
+            "iconUrl": "www.creedthoughts.gov.www'\'creedthoughts",
+            "copyRight": "Sometimes when I,m sick, or feeling blue, I drink vinegar. I like all kinds: balsamic, vodka, orange juice, leaves."
+        ]
+
+        guard let data = tradeMarkJSON.jsonString()?.data(using: .utf8) else {
+            fatalError("Bad JSON for trademark")
+        }
+
+        return data
+    }
+
+    //MARK:- BookingItemDetails
+    static func bookingItemDetailsData() -> Data {
+        let attribute1: MockJSON = [
+            "label": "Excludes",
+            "value": "Special exhibitions"
+        ]
+
+        let attribute2: MockJSON = [
+            "label": "Pre purchase",
+            "value": "Free entry for kids 0-3, 1 Meeseks per visitor, and Indigenous Peoples. Free entry on Tuesdays or when Ruben is sleeping"
+        ]
+
+        let contactInfo: MockJSON = [
+            "name": "Dr. Xenon Bloom",
+            "email": "xenon@bloom.com",
+            "website": "https://www.anatomyPark.com/contact",
+            "phones": nil
+        ]
+
+        let trademark: MockJSON = [
+            "iconUrl": "http:www.richSanchez.com/rick.jpg",
+            "copyRight": "Wubba lubba dub dub"
+        ]
+
+        let supplier: MockJSON = [
+            "name": "Rick Sanchez",
+            "trademark": trademark
+        ]
+
+        let exchangeRates: MockJSON = [
```

Comment:
It would be much easier to have these mocks normalized and available for future mocked responses as well. this is already the second time using the same mocked data

Reply:
This is just a suggestion

Reply:
Normalized these